### ISSUE TITLE
AttributeError: 'float' object has no attribute '__pos__'

### ISSUE THREAD

**Issue by rudi-c** [dropbox/pyston#775]
Float object is missing the operator `__pos__` (see `float.cpp`)

```
>> 5.0.__pos__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>:
AttributeError: 'float' object has no attribute '__pos__'
```

While we're at it, there's a number of other operators on float that are missing - see https://docs.python.org/2/library/operator.html

**Comment by Daetalus** [comment]
Solved by #793

**Comment by kmod** [comment]
Nice :)

fyi if put something like `closes #775` in your commit message, then when that commit gets merged it will [automatically close](https://help.github.com/articles/closing-issues-via-commit-messages/) the relevant issue :)  (plus it's some nice documentation)
